src/careers/current-openings/index.html

```diff
@@ -37,7 +37,7 @@
 
     {% import '_macro-table-current-openings.html' as table_current_openings %}
 
-    <section class="block
+    <div class="block
```

Comment:
(I'll stop leaving that comment, but it applies several other places in this PR.)